omas/omas_utils.py

```diff
@@ -7,6 +7,213 @@
 from .omas_setup import __version__
 import sys
 
+# --------------------------------------------
+# OMAS uncertain quantities
+# --------------------------------------------
+# The uncertainty package is awesome but slow
+from uncertainties.unumpy import nominal_values as _orig_nominal_values
+from uncertainties.unumpy import std_devs as _orig_std_devs
+import uncertainties
+
+if not hasattr(uncertainties.unumpy, '_orig_nominal_values'):
+    uncertainties.unumpy._orig_nominal_values = _orig_nominal_values
+if not hasattr(uncertainties.unumpy, '_orig_std_devs'):
+    uncertainties.unumpy._orig_std_devs = _orig_std_devs
+
+
+def nominal_values(obj):
+    if isinstance(obj, ouarray):
+        return obj.nominal_value
+    else:
+        return uncertainties.unumpy._orig_nominal_values(obj)
+
+
+uncertainties.unumpy.nominal_values = nominal_values
```

Comment:
Does this monkey-patching imply a certain order of importing `omas` and `uncertainties`?

Reply:
This works like all other monkey-patches. In OMFIT for example, we should import omas early on, so after that, the monkey patched version of `nominal_values` and `std_devs` will be imported everywhere else.

Reply:
Sure, OMFIT will do it right, but do you want to check for `uncertainties` in `sys.modules` here in `omas`, and warn or error if already imported?